supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -72,6 +71,11 @@ final case class TesTask(jobDescriptor: BackendJobDescriptor,
     `type` = Option("FILE")
   )
 
+  // TES accepts a key/value pair in its backend parameters that specifies
+  // the directory to use for files related to this task.
+  val tesTaskPathPrefix : (String, Option[String]) = ("internal_path_prefix",
+    Option(tesPaths.callExecutionRoot.resolve("tes_task").pathAsString))
```

Comment:
We might want to elevate this TES working directory to be part of `TesJobPaths` so we can easily reference it from other places (ex. your upcoming work to include files in that dir in metadata). 

It's a little weird, though, because it's implementation-specific and not part of the TES spec, so we'd need to be careful to not blow up when it doesn't exist.